<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project1Class</class>
 <widget class="QMainWindow" name="Project1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>595</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>595</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>576</width>
      <height>355</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Процессы</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>182</number>
         </attribute>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Путь</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Состояние</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="buttonClose">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>300</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Завершить процесс</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Автозапуск</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="autoStartTableWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>182</number>
         </attribute>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Путь</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Время</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionDelete"/>
    <addaction name="actionAddToAutoStart"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionAddToAutoStart"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSetTime"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Project1.qrc">
     <normaloff>:/Project1/img/folder.png</normaloff>:/Project1/img/folder.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="Project1.qrc">
     <normaloff>:/Project1/img/clear.png</normaloff>:/Project1/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="Project1.qrc">
     <normaloff>:/Project1/img/cancel.png</normaloff>:/Project1/img/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionAddToAutoStart">
   <property name="icon">
    <iconset resource="Project1.qrc">
     <normaloff>:/Project1/img/check.png</normaloff>:/Project1/img/check.png</iconset>
   </property>
   <property name="text">
    <string>Добавить в автозапуск</string>
   </property>
  </action>
  <action name="actionSetTime">
   <property name="icon">
    <iconset resource="Project1.qrc">
     <normaloff>:/Project1/img/time.png</normaloff>:/Project1/img/time.png</iconset>
   </property>
   <property name="text">
    <string>Отложить автозапуск</string>
   </property>
   <property name="toolTip">
    <string>Отложить автозапуск</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Project1.qrc"/>
 </resources>
 <connections/>
</ui>
